packages/expo-notifications/ios/EXNotifications/Notifications/Emitter/EXNotificationsEmitter.m

```diff
@@ -0,0 +1,85 @@
+// Copyright 2018-present 650 Industries. All rights reserved.
+
+#import <EXNotifications/EXNotificationsEmitter.h>
+#import <EXNotifications/EXNotificationSerializer.h>
+#import <EXNotifications/EXNotificationCenterDelegate.h>
+
+#import <UMCore/UMEventEmitterService.h>
+
+static NSString * const onDidReceiveNotification = @"onDidReceiveNotification";
+static NSString * const onDidReceiveNotificationResponse = @"onDidReceiveNotificationResponse";
```

Comment:
same comment as on android - should we expose a TS listener for this event as well?